CalendarFXView/src/main/java/com/calendarfx/model/EntryEditAction.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2017 Dirk Lemmermann Software & Consulting (dlsc.com)
+ * <p>
+ * This file is part of CalendarFX.
+ */
+package com.calendarfx.model;
+
+/**
+ * class used for parameter of {@link com.calendarfx.view.DateControl#entryActionPolicy} functional interface
+ */
+public class EntryEditAction {
```

Comment:
Please rename to EntryEditParameter to stay consistent with other parameter classes.